main.go

```diff
@@ -85,24 +88,34 @@ func main() {
 		os.Exit(1)
 	}
 
-	cfg := config.Default()
+	var cfg unversioned.EraserConfig
+	v1alpha1Config := v1alpha1Config.Default()
 	if configFile != "" {
-		o, err := options.AndFrom(ctrl.ConfigFile().AtPath(configFile).OfKind(cfg))
+		type apiVersion struct {
+			APIVerison string `json:"apiVersion"`
+		}
+		o, err := options.AndFrom(ctrl.ConfigFile().AtPath(configFile).OfKind(v1alpha1Config))
```

Comment:
proposed: use this little struct to unmarshal just the APIVersion. Then Convert whatever is received to `unversioned` using the appropriate auto-generated function.